src/renderer/inc/IRenderData.hpp

```diff
@@ -64,6 +65,22 @@ namespace Microsoft::Console::Render
         virtual const bool IsGridLineDrawingAllowed() noexcept = 0;
 
         virtual std::vector<Microsoft::Console::Types::Viewport> GetSelectionRects() noexcept = 0;
+        virtual bool IsAreaSelected() const = 0;
+        virtual void ClearSelection() = 0;

```

Comment:
Did we ever file a follow-up task to clean up / move `ClearSelection`,  `SelectNewRegion` and `SearchForText` from `IRenderData`?

Reply:
#1992 . I'm adding a comment in to track more easily.